src/data.rs

```diff
@@ -4,8 +4,10 @@ pub mod diskstats;
 pub mod flamegraphs;
 #[cfg(target_arch = "aarch64")]
 pub mod grv_perf_events;
+pub mod intel_icelake_perf_events;
```

Comment:
this should go after line 8 below.

Reply:
Yeah. Changing this as it should only be compiled on x86. rustfmt moved these :( .